rskj-core/src/main/java/co/rsk/peg/BridgeStorageIndexKey.java

```diff
@@ -25,6 +25,7 @@ public enum BridgeStorageIndexKey {
 
     SVP_FUND_TX_HASH_UNSIGNED("svpFundTxHashUnsigned"),
     SVP_FUND_TX_HASH_SIGNED("svpFundTxHashSigned"),
+    SVP_SPEND_TX_HASH_UNSIGNED("svpSpendTxHashUnsigned"),
```

Comment:
Should we put the same name as the one that appears in the [RSKIP](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP419.md?plain=1#L222)?

Reply:
I prefer how you did it to be consistent with the naming convention of the rest of the entries.

Reply:
That's a good question, if we keep this name we need to update the RSKIP. Let's check with @julia-zack when she's back